### DISCUSSION TITLE
Custom listings of cross-refs

### DISCUSSION THREAD

**Discussion by jbrick55** [quarto-dev/quarto-cli#8153] [Feature Requests]
### Description

I am able to create listings of files (images, .txt files, .qmd files, etc..) in my Quarto website. What I'd like to do is create a listing of all examples (#exm-) or definitions (#defs-). Is this possible? I could not figure it out from this page https://quarto.org/docs/websites/website-listings-custom.html. Thanks in advance for any assistance.

**Comment by cscheid** [comment]
Unfortunately that's not possible yet (but it's definitely on our radar!).

**Reply by jbrick55** [reply]
Thanks!!
